Doc for Quable (#3085)
ezsystems/developer-documentation: docs/product_catalog/product_catalog.md

---

> description: Ibexa DXP provides product catalog capabilities for managing products, product types, variants, attributes, pricing, and catalogs.
> page_type: landing_page

Reviewer: This introduction to the total revamp of what was PIM (https://doc.ibexa.co/en/5.0/pim/pim/) worth being labeled as new.

```suggestion
description: Ibexa DXP provides product catalog capabilities for managing products, product types, variants, attributes, pricing, and catalogs.
page_type: landing_page
month_change: true
```

I like this new https://ez-systems-developer-documentation--3085.com.readthedocs.build/en/3085/product_catalog/product_catalog/ by the way.

---

# Product Catalog

The Product Catalog provides comprehensive capabilities for managing products offered in your digital commerce experience, including their specifications, pricing, and organization.

[[= product_name =]] offers robust product catalog infrastructure that can be used standalone.

> You can also use [Quable Product Information Management (PIM)](quable/quable.md) that's fully integrated into the [[= product_name_base =]] ecosystem, or the [Remote PIM](add_remote_pim_support.md) to add integration with any external PIM system.

Reviewer: ```suggestion
You can also use [[[= pim_product_name =]] Product Information Management (PIM)](quable/quable.md) that's fully integrated into the [[= product_name_base =]] ecosystem, or the [Remote PIM](add_remote_pim_support.md) to add integration with any external PIM system.
```

Reviewer: This variable doesn't seem alway used. Did you create a vale rule for it?

Author: Please see https://github.com/ibexa/vale-styles/pull/38

---
description: Ibexa DXP provides product catalog capabilities for managing products, product types, variants, attributes, pricing, and catalogs.
page_type: landing_page
month_change: true
---

> # Product Catalog

Reviewer 2: As we've had "product catalogs" that are now "catalogs", and PIM that is now "product catalog", the change may be confusing, and calls for a blog post that announces the updated nomenclature and provides justification.

The Product Catalog provides comprehensive capabilities for managing products offered in your digital commerce experience, including their specifications, pricing, and organization.

[[= product_name =]] offers robust product catalog infrastructure that can be used standalone.

> You can also use [[[= pim_product_name =]] Product Information Management (PIM)](/product_catalog/quable/quable.md) that's fully integrated into the [[= product_name_base =]] ecosystem, or the [Remote PIM](add_remote_pim_support.md) to add integration with any external PIM system.

Reviewer 2: ```suggestion
You can also use [[[= pim_product_name =]] Product Information Management (PIM)](/product_catalog/quable/quable.md) add-on that fully integrates into the [[= product_name_base =]] ecosystem, or the [Remote PIM](add_remote_pim_support.md) to add integration with any external PIM system.
```
